src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/LayoutTestControl/MeasureArrangeCounter.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.LayoutTestControl
+{
+	public partial class MeasureArrangeCounter : ContentControl
+	{
+		public MeasureArrangeCounter()
+		{
+			HorizontalAlignment = HorizontalAlignment.Stretch;
+			HorizontalContentAlignment = HorizontalAlignment.Stretch;
+			VerticalAlignment = VerticalAlignment.Stretch;
+			VerticalContentAlignment = VerticalAlignment.Stretch;
+		}
+		public string MeasureCount
+		{
+			get { return (string)GetValue(MeasureCountProperty); }
+			set { SetValue(MeasureCountProperty, value); }
+		}
+
+		public static readonly DependencyProperty MeasureCountProperty =
+			DependencyProperty.Register("MeasureCount", typeof(string), typeof(MeasureArrangeCounter), new PropertyMetadata("."));
+
+		public string ArrangeCount
+		{
+			get { return (string)GetValue(ArrangeCountProperty); }
+			set { SetValue(ArrangeCountProperty, value); }
+		}
+
+		public static readonly DependencyProperty ArrangeCountProperty =
+			DependencyProperty.Register("ArrangeCount", typeof(string), typeof(MeasureArrangeCounter), new PropertyMetadata("."));
+
+
+		public void UpdateCounts()
+		{
+			MeasureCount = $"Measure : {_measureCount}";
+			ArrangeCount = $"Arrange : {_arrangeCount}";
+		}
+
+		int _measureCount = 0;
+		int _arrangeCount = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to '_arrangeCount', the compiler will do that for you.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826544)